apps/videos-admin/src/app/(dashboard)/videos/[videoId]/citation/[citationId]/delete/page.tsx

```
@@ -0,0 +1,75 @@
+        closeButton: true
+      }}
+      dialogAction={{
+        onSubmit: handleDeleteCitation,
+        submitLabel: 'Delete',
+        closeLabel: 'Cancel'
+      }}
+      loading={deleteLoading}
+    >
+      Are you sure you want to delete this Bible citation? This action cannot be
+      undone.
+    </Dialog>
```

ESLint

[error] disallow literal string: <Dialog
      open={true}
      onClose={() => router.push(`/videos/${videoId}`, { scroll: false })}
      dialogTitle={{
        title: 'Delete Bible Citation',
        closeButton: true
      }}
      dialogAction={{
        onSubmit: handleDeleteCitation,
        submitLabel: 'Delete',
        closeLabel: 'Cancel'
      }}
      loading={deleteLoading}
    >
      Are you sure you want to delete this Bible citation? This action cannot be
      undone.
    </Dialog>

(i18next/no-literal-string)